buildenv/jenkins/JenkinsfileBase

```diff
@@ -753,6 +770,18 @@ def post(output_name) {
 	}
 }
 
+def getjobStatus(){
+	steps {
+		script {
+			def job = jenkins.model.Jenkins.instance.getItemByFullName("Machine-Reservation")
+			def result = job.getLastBuild().getResult().toString()
```

Comment:
`getLastBuild` may not be the build that you triggered. The correct way is to get job info when you are triggering the job (line 611). For this PR, let's just remove this getjobStatus() for now.